Casks/t/timescribe.rb

```diff
@@ -0,0 +1,23 @@
+cask "timescribe" do
+  arch arm: "arm64", intel: "x64"
+
+  version "1.2.1"
+  sha256 :no_check
+
+  url "https://timescribe.app/download/macos/#{arch}"
```

Comment:
Is there a reason for not using GitHub here to allow a versioned download `url`? This would let us also have a checksum for added security protections.

```suggestion
  url "https://github.com/WINBIGFOX/TimeScribe/releases/download/v#{version}/TimeScribe-#{version}-#{arch}.zip,
      verified: "github.com/WINBIGFOX/TimeScribe/"